PowerPointLabs/PowerPointLabs/CropLab/CropToSame.cs

```diff
@@ -0,0 +1,169 @@
+﻿using System;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Windows.Forms;
+
+using Office = Microsoft.Office.Core;
+using PowerPoint = Microsoft.Office.Interop.PowerPoint;
+
+namespace PowerPointLabs.CropLab
+{
+    public class CropToSame
+    {
+#pragma warning disable 0618
```

Comment:
new codes should not disable this.
and try not to copy/paste `CropToShape` codes